### DISCUSSION TITLE
model do not apear on open webui

### DISCUSSION THREAD

**Discussion by kshabanaa** [open-webui/open-webui#1823] [General]
# Bug Report
i have llama 3 on ollama but when i search for llama 3 in open webui i didnot find llama 3
![Screenshot 2024-04-28 134412](https://github.com/open-webui/open-webui/assets/146099305/37c337d8-89da-459f-9edd-5e0f3126a851)

![Screenshot 2024-04-28 135421](https://github.com/open-webui/open-webui/assets/146099305/b31ca71a-2edf-489f-af18-71783b97dbf3)

**Bug Summary:**
[ I have llama 3 on ollama cli but it does not apear for me on open webui]

- **Operating System:** [e.g., Windows 11]
- **Browser [Chrome]
- [x] i think that I am on the latest version of both Open WebUI and Ollama.

**Comment by M-ax-G** [comment]
[Solved] I completely removed docker; reinstalled via get-docker.sh;
pulled Open WebUIimage with `docker run -d --network=host -v open-webui:/app/backend/data -e OLLAMA_BASE_URL=http://127.0.0.1:11434 --name open-webui --restart always ghcr.io/open-webui/open-webui:main`
--> Now it works.

---

I seem to have the same issue... cannot select a model despite being connected to CLI install of ollama.

On Linux Mint 21.3 Virginia
... with ollama version is 0.1.33 pulled via `curl -fsSL https://ollama.com/install.sh | sh`
resulting in: `>>> The Ollama API is now available at 127.0.0.1:11434`
... which it is.
Then ran: `ollama run llama3`
Using the prompt and asking a question works.

Then installed Open WebUI like so:
`docker run -d --network=host -v open-webui:/app/backend/data -e OLLAMA_BASE_URL=http://127.0.0.1:11434 --name open-webui --restart always ghcr.io/open-webui/open-webui:main`

I signed up and the WebUI seems to work.

I checked the connection URL, which says server connection verified, when using: `http://localhost:8080`

I then "Select Model", which is empty, and typing llama3 does not yield a result.

Now I am at a loss. Any hints appreciated.
